# Datasets by Category

## Allergen Identification & Prediction

| Name | Description | Source | License | Tags | Access |
|------|-------------|--------|---------|------|--------|
| STITCH | A database that itegrates known and predicted interactions between chemicals and proteins, combining evidence from experiments, databases, text mining and prediction algorithms | [http://stitch.embl.de/cgi/download.pl?UserId=o7OnPFVV3JJ4&sessionId=e44tciEEXzEc](http://stitch.embl.de/cgi/download.pl?UserId=o7OnPFVV3JJ4&sessionId=e44tciEEXzEc) |  | Allergen Identification, Allergenicity Assessment | Open source |
| [Allermatch](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC522748/) | Webtool for standardized allergenicity prediction according to FAO/WHO Codex alimentarius guidelines using sliding window approach | [http://allermatch.org](http://allermatch.org) | [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC522748/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC522748/) | Allergenicity Assessment | Open source |
| AllerHunter | Computational tool for allergen prediction with internal and external validation achieving MCC 0.738 on external dataset | Contact authors | Multiple publications | Allergenicity Assessment | Gated |
| [NetAllergen](https://academic.oup.com/bioinformaticsadvances/article/3/1/vbad151/7319372?login=false) | A curated database of IgE-inducing allergens based on AllergenOnline, carefully removed allergen redundancy with a novel protein partitioning pipeline, and developed a new allergen prediction method, introducing MHC presentation propensity as a novel feature. | [https://services.healthtech.dtu.dk/services/NetAllergen-1.0/](https://services.healthtech.dtu.dk/services/NetAllergen-1.0/) | [https://academic.oup.com/bioinformaticsadvances/article/3/1/vbad151/7319372?login=false](https://academic.oup.com/bioinformaticsadvances/article/3/1/vbad151/7319372?login=false) | Allergen Identification, Allergenicity Assessment | Unknown |
| [Akkermansia muciniphila exacerbates food allergy in fibre-deprived mice](https://www.nature.com/articles/s41564-023-01464-1) | Study on alteration of mice gut microbioma, focusing on Akkermansia muciniphila. | [https://www.ebi.ac.uk/ena/browser/view/PRJEB53451](https://www.ebi.ac.uk/ena/browser/view/PRJEB53451) | [https://www.nature.com/articles/s41564-023-01464-1](https://www.nature.com/articles/s41564-023-01464-1) | Allergen Identification, Allergenicity Assessment | Open source |
| [CHILD cohort](https://www.nature.com/articles/s41467-023-40336-4#data-availability) | Multi-omics; microbiome maturation predicts allergic disease |  | [https://www.nature.com/articles/s41467-023-40336-4#data-availability](https://www.nature.com/articles/s41467-023-40336-4#data-availability) | Allergen Identification, Allergenicity Assessment | Gated |
| [WHO/IUIS Allergen Nomenclature Database](https://pubmed.ncbi.nlm.nih.gov/29625844/) | The WHO/IUIS Allergen Nomenclature is the authoritative system for naming allergenic proteins, approved by the World Health Organization and International Union of Immunological Societies. Established in 1984, this sub-committee maintains a unique, systematic nomenclature based on the Linnaean taxonomy for proteins causing IgE-mediated allergic reactions, supporting global consistency in allergen research and publication | [https://allergen.org/](https://allergen.org/) | [https://pubmed.ncbi.nlm.nih.gov/29625844/](https://pubmed.ncbi.nlm.nih.gov/29625844/) | Text Mining | Open source |


## Computational Method Development

| Name | Description | Source | License | Tags | Access |
|------|-------------|--------|---------|------|--------|
| [DAVIS (DrugTarget)](https://pubmed.ncbi.nlm.nih.gov/24723570/) | Drug-target affinity dataset containing Kd values for 68 drugs and 379 protein targets, widely used for benchmarking drug-target interaction prediction models | [https://staff.cs.utu.fi/~aatapa/data/DrugTarget/](https://staff.cs.utu.fi/~aatapa/data/DrugTarget/) | [https://pubmed.ncbi.nlm.nih.gov/24723570/](https://pubmed.ncbi.nlm.nih.gov/24723570/) | Drug Design, Target Interaction | Open source |
| [nabla²DFT](https://pubs.rsc.org/en/content/articlelanding/2022/CP/D2CP03966D) | nabla²DFT is a large-scale dataset and benchmark in computational quantum chemistry that is designed to support machine learning models for predicting molecular electronic structure properties. Per molecule, it contains quantum-level properties like total electronic energy, DFT Hamilton matrices, forces, overlap matrices, etcetera. In addition to the data, it also contains benchmark tasks. Can be used to train neural network potentials. | [https://github.com/AIRI-Institute/nablaDFT/tree/1.0](https://github.com/AIRI-Institute/nablaDFT/tree/1.0) | [https://pubs.rsc.org/en/content/articlelanding/2022/CP/D2CP03966D](https://pubs.rsc.org/en/content/articlelanding/2022/CP/D2CP03966D) | Structural Analysis, Drug Design | Open source |
| QM Datasets | Benchmarks quantum chemistry datasets of small organic molecules (<=9 heavy atoms) where molecular properties have been computed via quantum chemistry. WIDELY USED for molecular property prediction | [https://quantum-machine.org/datasets/](https://quantum-machine.org/datasets/) |  | Property Prediction, Text Mining | Open source |
| [QDπ Dataset](https://www.nature.com/articles/s41597-025-04972-3) | The QDπ dataset enables creation of flexible target loss functions for neural network training relevant to drug discovery, including information-dense data sets of relative conformational energies and barriers, intermolecular interactions, tautomers and relative protonation energies of drug-like compounds and biomolecular fragments. Useful for training universal machine learning potentials (MLPs). | [https://zenodo.org/records/14970869](https://zenodo.org/records/14970869) | [https://www.nature.com/articles/s41597-025-04972-3](https://www.nature.com/articles/s41597-025-04972-3) | Property Prediction, Text Mining | Open source |
| [AlgPred 2.0 Dataset](https://pubmed.ncbi.nlm.nih.gov/33201237/) | Large-scale dataset with 10075 allergens and 10075 non-allergens plus 10451 validated IgE epitopes for machine learning | [https://webs.iiitd.edu.in/raghava/algpred2/](https://webs.iiitd.edu.in/raghava/algpred2/) | [https://pubmed.ncbi.nlm.nih.gov/33201237/](https://pubmed.ncbi.nlm.nih.gov/33201237/) | Allergenicity Assessment, Cross-Reactivity Modeling, Drug Design | Open source |
| Allergen Chip data challenge | The goal of the competition was to develop Machine Learning models that can predict the presence and severity of an allergic disease based on this personalized profile. The dataset has been constructed from data of more than 4,000 patients includes tabular data associated with image files. | [https://github.com/Trustii-team/AllergenChip](https://github.com/Trustii-team/AllergenChip) |  | Risk Stratification, Severity Assessment | Gated |
| [TIP Dataset](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0283141) | Tolerance Induction Program dataset containing data from 530 pedriatic patients. From "Food anaphylaxis diagnostic marker compilation in machine learning design and validation" | [https://github.com/TPIRC/ai_paper_2022](https://github.com/TPIRC/ai_paper_2022) | [https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0283141](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0283141) | Risk Stratification, Severity Assessment | Open source |


## Cross-Reactivity Analysis

| Name | Description | Source | License | Tags | Access |
|------|-------------|--------|---------|------|--------|
| [Allergen30](https://link.springer.com/article/10.1007/s12161-022-02353-9) | Dataset containing structural and sequence information for 30 major allergen families to support allergenicity prediction and cross-reactivity analysis | [https://data.mendeley.com/datasets/9ygs9vhnpw/1](https://data.mendeley.com/datasets/9ygs9vhnpw/1) | [https://link.springer.com/article/10.1007/s12161-022-02353-9](https://link.springer.com/article/10.1007/s12161-022-02353-9) | Allergenicity Assessment, Allergen Identification, Structural Analysis | Open source |
| [AllFam](https://pubmed.ncbi.nlm.nih.gov/18395549/) | Database classifying allergens into 134 protein families based on WHO/IUIS and AllergenOnline data with Pfam definitions | [https://www.meduniwien.ac.at/allfam/](https://www.meduniwien.ac.at/allfam/) | [https://pubmed.ncbi.nlm.nih.gov/18395549/](https://pubmed.ncbi.nlm.nih.gov/18395549/) | Structural Analysis, Cross-Reactivity Modeling | Open source |
| [IEDB Analysis Resource](https://academic.oup.com/nar/article/47/W1/W502/5494780) | Companion to IEDB providing computational tools for B and T cell epitope prediction including MHC binding predictions | [http://tools.iedb.org/](http://tools.iedb.org/) | [https://academic.oup.com/nar/article/47/W1/W502/5494780](https://academic.oup.com/nar/article/47/W1/W502/5494780) | Epitope Mapping | Open source |
| [AllergenAI](https://pmc.ncbi.nlm.nih.gov/articles/PMC11230160/) | Allergenicity prediction based on protein sequences. Processed data from SDAP 2.0, COMPARE, and AlgPred 2 | [https://compbio.uth.edu/AllergenAI/](https://compbio.uth.edu/AllergenAI/) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC11230160/](https://pmc.ncbi.nlm.nih.gov/articles/PMC11230160/) | Allergen Identification, Allergenicity Assessment | Open source |
| Allergen Family Database | A curated database that classifies known allergens into protein families to support analysis of allergenicity and cross-reactivity across sources and exposure routes. It integrates entries from WHO/IUIS Allergen Nomenclature and AllergenOnline with Pfam domain annotations, providing family-level pages with biochemical descriptions, allergological significance, and links to primary records and references. | [https://www.meduniwien.ac.at/allfam/](https://www.meduniwien.ac.at/allfam/) |  | Allergen Identification, Allergenicity Assessment, Drug Design, Target Interaction, Structural Analysis | Open source |
| Alleropedia Database for Allergens | The Alleropedia database is a comprehensive metadatabase consolidating 13,146 allergen records from six freely accessible sources, including major allergen databases like COMPARE, AllergenOnline, WHO/IUIS, and Allergome. It offers a user-friendly web interface and additional features such as data integration with sources like NCBI, facilitating easy access, analysis, and navigation of allergen-related information for researchers and clinician | [https://github.com/maitreyeepaliwal/Alleropedia-Database-for-Allergens](https://github.com/maitreyeepaliwal/Alleropedia-Database-for-Allergens) |  | Allergen Identification, Allergenicity Assessment, Cross-Reactivity Modeling, Epitope Mapping, Structural Analysis | Open source |


## Drug & Immunotherapy Development

| Name | Description | Source | License | Tags | Access |
|------|-------------|--------|---------|------|--------|
| [SDAP 2.0](https://pmc.ncbi.nlm.nih.gov/articles/PMC10509899/) | SDAP is a Web server that integrates a database of allergenic proteins with various computational tools that can assist structural biology studies related to allergens. SDAP is an important tool in the investigation of the cross-reactivity between known allergens, in testing the FAO/WHO allergenicity rules for new proteins, and in predicting the IgE-binding potential of genetically modified food proteins. | [https://fermi.utmb.edu/](https://fermi.utmb.edu/) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC10509899/](https://pmc.ncbi.nlm.nih.gov/articles/PMC10509899/) | Drug Design, Structural Analysis, Epitope Mapping, Cross-Reactivity Modeling | Open source |
| [QSAR Database](https://qsardb.org/) | QsarDB is a smart repository for (Q)SAR/QSPR models and datasets, providing access to peer-reviewed quantitative structure-activity relationship models | [https://qsardb.org/](https://qsardb.org/) | [https://qsardb.org/](https://qsardb.org/) | Property Prediction, Drug Design | Open source |
| [e-Drug3D Database](https://pmc.ncbi.nlm.nih.gov/articles/PMC5846051/) | Three-dimensional database of drug-like compounds and their molecular conformations for structure-based drug design applications | [https://zenodo.org/records/17063565](https://zenodo.org/records/17063565) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC5846051/](https://pmc.ncbi.nlm.nih.gov/articles/PMC5846051/) | Drug Design, Target Interaction | Open source |
| [Stanford Drug Data (Effects & Interactions)](https://pmc.ncbi.nlm.nih.gov/articles/PMC3382018/) | Comprehensive dataset of drug effects and drug-drug interactions compiled from clinical data and pharmacological studies | [https://purl.stanford.edu/zq918jm7358/version/1](https://purl.stanford.edu/zq918jm7358/version/1) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC3382018/](https://pmc.ncbi.nlm.nih.gov/articles/PMC3382018/) | Drug Design | Open source |
| DrugCentral | Open-access online drug information repository covering over 4950 drugs with structural, physicochemical, and pharmacological details to support drug discovery and repositioning | [https://drugcentral.org/download](https://drugcentral.org/download) |  | Drug Design, Target Interaction | Open source |
| [MedKG](https://link.springer.com/article/10.1007/s11030-025-11164-z) | Comprehensive medical knowledge graph integrating data from 35 authoritative sources with 34 node types and 79 relationships for precision medicine and drug discovery | [https://github.com/chemplusx/MedKG](https://github.com/chemplusx/MedKG) | [https://link.springer.com/article/10.1007/s11030-025-11164-z](https://link.springer.com/article/10.1007/s11030-025-11164-z) | Drug Design, Target Interaction, Treatment Planning | Open source |
| [PDBBind+](https://arxiv.org/html/2308.09639v2) | Enhanced version of PDBBind database providing protein-ligand binding affinity data with refined experimental measurements and structural information | [https://www.pdbbind-plus.org.cn/download](https://www.pdbbind-plus.org.cn/download) | [https://arxiv.org/html/2308.09639v2](https://arxiv.org/html/2308.09639v2) | Target Interaction, Drug Design | Open source |
| [Human Metabolome Database (HMDB)](https://pmc.ncbi.nlm.nih.gov/articles/PMC1899095/) | Freely available electronic database containing detailed information about 220,945 small molecule metabolites found in the human body for metabolomics and biomarker discovery | [https://www.hmdb.ca/downloads](https://www.hmdb.ca/downloads) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC1899095/](https://pmc.ncbi.nlm.nih.gov/articles/PMC1899095/) | Drug Design, Target Interaction | Open source |
| [Therapeutic Target Database (TTD)](https://academic.oup.com/nar/article/52/D1/D1465/7275004) | Database providing information about known therapeutic protein and nucleic acid targets, targeted diseases, pathway information, and corresponding drugs | [https://db.idrblab.net/ttd/full-data-download](https://db.idrblab.net/ttd/full-data-download) | [https://academic.oup.com/nar/article/52/D1/D1465/7275004](https://academic.oup.com/nar/article/52/D1/D1465/7275004) | Drug Design, Treatment Planning, Target Interaction | Open source |
| [QCML Dataset](https://www.nature.com/articles/s41597-025-04720-7) | Quantum chemistry reference dataset with 33.5 m. DFT calculations and 14.7 billion semi empirical entries. It covers small molecules (up to 8 heavy atoms) and provide a wide variety of computed molecular properties | [https://zenodo.org/records/14859804](https://zenodo.org/records/14859804) | [https://www.nature.com/articles/s41597-025-04720-7](https://www.nature.com/articles/s41597-025-04720-7) | Drug Design, Target Interaction | Open source |
| [M3-20M: Multi-Modal Molecular Dataset](https://arxiv.org/html/2412.06847v2) | M3-20M is an extensive multi-modal molecular dataset containing over 20 million molecules with 1D, 2D, and 3D molecular representations, physicochemical properties, and text descriptions. It supports AI-driven drug discovery, molecular property prediction, lead optimization, and drug-target interaction modeling across various applications, including allergy-related therapeutic design. | [https://huggingface.co/datasets/Alex99Gsy/M-3_Multi-Modal-Molecule](https://huggingface.co/datasets/Alex99Gsy/M-3_Multi-Modal-Molecule) | [https://arxiv.org/html/2412.06847v2](https://arxiv.org/html/2412.06847v2) | Target Interaction, Property Prediction, Drug Design | Open source |
| [SAIR Dataset](https://go.sandboxaq.com/rs/175-UKR-711/images/sair_paper.pdf) | The SAIR dataset is a massive repository containing over one million protein-ligand 3D cofolded structures paired with experimental binding affinity measurements (e.g., IC50). It supports AI-driven drug discovery by enabling prediction of molecular binding potency and facilitating the design and optimization of new therapeutic compounds targeting allergenic proteins and other disease-related targets. | [https://huggingface.co/datasets/SandboxAQ/SAIR](https://huggingface.co/datasets/SandboxAQ/SAIR) | [https://go.sandboxaq.com/rs/175-UKR-711/images/sair_paper.pdf](https://go.sandboxaq.com/rs/175-UKR-711/images/sair_paper.pdf) | Target Interaction, Structural Analysis, Property Prediction, Drug Design | Gated |
| TDC | Curated collection of AI (ready) datasets and tasks in the therapeutic pipeline, with consistent splits and evals | [https://tdcommons.ai/](https://tdcommons.ai/) |  | Drug Design, Target Interaction | Open source |
| [RxRx3](https://arxiv.org/abs/2503.20158) | Cell imaging phenomics dataset, that provides a map for biology for ML methods, including knockouts small molecule perturbations and embeddings | [https://www.recursion.com/news/accelerating-ai-drug-discovery-with-open-source-datasets](https://www.recursion.com/news/accelerating-ai-drug-discovery-with-open-source-datasets) | [https://arxiv.org/abs/2503.20158](https://arxiv.org/abs/2503.20158) | Drug Design, Target Interaction | Open source |
| Dorothea | Gene regulatory network of signed TF-> target interactions (human/mouse) with confidence levels | [https://archive.ics.uci.edu/ml/datasets/dorothea](https://archive.ics.uci.edu/ml/datasets/dorothea) |  | Drug Design, Target Interaction | Open source |
| Simulated Allergen Immunotherapy Trials Dataset | Simulated datasets implementing an enchanced three stage trial design for allergen immunotherapy (AIT). It captures realistic features like corssover, discontinuation and staged enrollment | [https://figshare.com/articles/dataset/Simulated_datasets_for_enhanced_three-stage_design_for_allergen_immunotherapy_trials/23638965](https://figshare.com/articles/dataset/Simulated_datasets_for_enhanced_three-stage_design_for_allergen_immunotherapy_trials/23638965) |  | Drug Design, Target Interaction | Open source |
| IUPHAR Pharmacology Datasets | Pharmacology data curated from experts that links drug/ligand information to molecular targets | [https://www.guidetopharmacology.org/download.jsp](https://www.guidetopharmacology.org/download.jsp) |  | Drug Design, Target Interaction | Open source |
| Enamine REAL Database | Massive virtual libary of synthesizable compunds and enumerated subsets for large scale virtual screening and hit expansion | [https://gist.github.com/matteoferla/b1eee8656079d006835f2d8dc159fbb5](https://gist.github.com/matteoferla/b1eee8656079d006835f2d8dc159fbb5) |  | Drug Design, Target Interaction | Open source |
| [Quantum Chemistry Database with Ground- and Excited-State (QCDGE) Dataset](https://www.nature.com/articles/s41597-024-03788-x) | more than 400k small organic molecules (<=10 heavy atoms) for which both ground state and excited state quantum chemical properties have already been computed | [https://springernature.figshare.com/collections/QCDGE_database_Quantum_Chemistry_Database_with_Ground-_and_Excited-State_Properties/7259125/1](https://springernature.figshare.com/collections/QCDGE_database_Quantum_Chemistry_Database_with_Ground-_and_Excited-State_Properties/7259125/1) | [https://www.nature.com/articles/s41597-024-03788-x](https://www.nature.com/articles/s41597-024-03788-x) | Drug Design, Target Interaction | Open source |
| Probes & Drugs Datasets | The Probes & Drugs (P&D) portal is a comprehensive resource integrating high-quality bioactive compound sets for analysis and comparison, focusing on chemical probes and drugs. It includes compound data from multiple sources, provides expert scoring based on potency and selectivity, and offers standardized compound forms to unify data. The portal supports research by tagging probes, scoring probe-likeness, and highlighting structural alerts for compound reliability | [https://www.probes-drugs.org/download](https://www.probes-drugs.org/download) |  | Target Interaction | Open source |
| [AllerBase](https://academic.oup.com/database/article/doi/10.1093/database/bax066/4157991) | Comprehensive allergen knowledgebase integrating data from multiple sources with extensive experimental validation and IgE epitope data | [http://algpred.tu-bs.de/allerbase/](http://algpred.tu-bs.de/allerbase/) | [https://academic.oup.com/database/article/doi/10.1093/database/bax066/4157991](https://academic.oup.com/database/article/doi/10.1093/database/bax066/4157991) | Cross-Reactivity Modeling, Allergenicity Assessment, Drug Design | Open source |
| Simulated AIT Trials Dataset | Simulated datasets with enhanced three-stage trial design for allergen immunotherapy capturing realistic features like crossover and discontinuation | [https://figshare.com/articles/dataset/Simulated_datasets_for_enhanced_three-stage_design_for_allergen_immunotherapy_trials/23638965](https://figshare.com/articles/dataset/Simulated_datasets_for_enhanced_three-stage_design_for_allergen_immunotherapy_trials/23638965) |  | Treatment Planning, Drug Design | Open source |
| [Food Anaphylaxis ML Dataset (TIP)](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0283141) | Dataset from Tolerance Induction Program with 530 juvenile patients featuring 241 allergy assays per patient achieving 95.2% recall for peanut anaphylaxis prediction | Contact authors | [https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0283141](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0283141) | Drug Design, Treatment Planning | Gated |
| [Food Allergy Risk Stratification Dataset](https://onlinelibrary.wiley.com/doi/10.1111/all.15839) | EMR-based dataset with 4077 children with food allergies and 95686 controls for predicting FA development with AUC 0.80 | Contact authors | [https://onlinelibrary.wiley.com/doi/10.1111/all.15839](https://onlinelibrary.wiley.com/doi/10.1111/all.15839) | Treatment Planning | Gated |
| Enamine REAL Database | Massive virtual library of synthesizable compounds and enumerated subsets for large-scale virtual screening and hit expansion | [https://gist.github.com/matteoferla/b1eee8656079d006835f2d8dc159fbb5](https://gist.github.com/matteoferla/b1eee8656079d006835f2d8dc159fbb5) |  | Drug Design, Target Interaction | Open source |
| [AllerCatPro 2.0](https://pubmed.ncbi.nlm.nih.gov/35157755/) | Tool predicting allergenicity using amino acid sequence and 3D structure similarity with database of 4979 allergens 162 mild allergenic proteins and 165 autoimmune allergens | [https://allercatpro.bii.a-star.edu.sg/](https://allercatpro.bii.a-star.edu.sg/) | [https://pubmed.ncbi.nlm.nih.gov/35157755/](https://pubmed.ncbi.nlm.nih.gov/35157755/) | Allergenicity Assessment, Drug Design | Open source |
| [AllerTOP v1.1](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S6-S4) | First alignment-free server for in silico prediction of allergens based on physicochemical properties of proteins, achieving 94% sensitivity in allergen prediction | [https://ddg-pharmfac.net/allertop/cite/](https://ddg-pharmfac.net/allertop/cite/) | [https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S6-S4](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S6-S4) | Structural Analysis, Drug Design, Allergenicity Assessment, Allergen Identification | Open source |
| FARE Food Allergy Research | The Data Coordinating Center will support critical FARE Clinical Network activities for the design, development, execution, monitoring, and analysis of translational research. | [https://research.foodallergy.org/#_ga=2.222605656.279038653.1759159933-230927819.1759159933](https://research.foodallergy.org/#_ga=2.222605656.279038653.1759159933-230927819.1759159933) |  | Risk Stratification, Severity Assessment, Symptom Analysis, Treatment Planning | Gated |
| [Allergy dataset](https://pubmed.ncbi.nlm.nih.gov/27542726/) | Dataset supporting the conclusions for article: "The epidemiologic characteristics of healthcare provider-diagnosed eczema, asthma, allergic rhinitis, and food allergy in children: a retrospective cohort study" by Hill et al. | [https://zenodo.org/records/44529](https://zenodo.org/records/44529) | [https://pubmed.ncbi.nlm.nih.gov/27542726/](https://pubmed.ncbi.nlm.nih.gov/27542726/) | Risk Stratification, Ingredient Analysis, Treatment Planning, Product Development, Genetic Analysis | Open source |
| [Allergome](https://www.jacionline.org/article/S0091-6749(04)03615-2/fulltext) | A comprehensive, curated platform documenting allergenic molecules and their sources across all taxa and exposure routes, with monographs, literature integration, and tools tailored for clinicians and researchers in allergy and immunology. | [https://www.allergome.org/](https://www.allergome.org/) | [https://www.jacionline.org/article/S0091-6749(04)03615-2/fulltext](https://www.jacionline.org/article/S0091-6749(04)03615-2/fulltext) | Allergenicity Assessment, Drug Design, Allergen Identification | Open source |
| [GWAS Database](https://pmc.ncbi.nlm.nih.gov/articles/PMC2639349/) | The NHGRI-EBI GWAS Catalog is a curated, standardized repository of human genome-wide association study results, offering hundreds of thousands of variant–trait associations and tens of thousands of full summary-statistics datasets suitable for downstream analyses like meta-analysis and fine-mapping. Because it indexes GWAS signals across many immune and barrier-function traits and includes loci implicated in food allergy (e.g., HLA, FLG, SERPINB cluster), it is directly usable to query, aggregate, and reanalyze genetic associations relevant to food allergies and specific allergens such as peanut | [https://www.ebi.ac.uk/gwas/](https://www.ebi.ac.uk/gwas/) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC2639349/](https://pmc.ncbi.nlm.nih.gov/articles/PMC2639349/) | Ingredient Analysis, Product Development | Open source |
| COMPARE | The COMPARE database provides an annually updated, peer-reviewed collection of clinically relevant protein sequences of allergens, along with tools for aligning and assessing the allergenic potential of novel proteins based on established regulatory guidelines | [https://comparefasta.comparedatabase.org/](https://comparefasta.comparedatabase.org/) |  | Allergen Identification, Cross-Reactivity Modeling, Allergenicity Assessment, Epitope Mapping, Risk Stratification, Ingredient Analysis, Target Interaction, Severity Assessment | Open source |
| Open Food Facts | The Open Food Facts database is a large, publicly accessible collection of detailed product information including ingredients, nutrition, and labeling, available in multiple data formats with open licenses for broad reuse in food transparency and research.This dataset mainly supports food-related analyses including allergen detection, labeling validation, chemical and nutritional content analysis, and research into hypoallergenic or alternative food products | [https://world.openfoodfacts.org/data](https://world.openfoodfacts.org/data) |  | Ingredient Analysis, Labeling Compliance, Property Prediction, Treatment Planning, Product Development, Alternative Ingredients | Open source |
| [IEDB (Immune Epitope Database)](https://pmc.ncbi.nlm.nih.gov/articles/PMC3042621) | Comprehensive database containing over 1.6 million immune epitopes including antibody and T cell epitopes for allergens, with analysis and prediction tools | [https://www.iedb.org/](https://www.iedb.org/) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC3042621](https://pmc.ncbi.nlm.nih.gov/articles/PMC3042621) | Epitope Mapping, Target Interaction, Drug Design, Product Development | Open source |


## Food Product Development & Safety

| Name | Description | Source | License | Tags | Access |
|------|-------------|--------|---------|------|--------|
| [Allergen30](https://link.springer.com/article/10.1007/s12161-022-02353-9) | More than 6,000 images of 30 commonly used food items which can cause an adverse reaction within a human body. The goal is building a robust detection model that can assist people in avoiding possible allergic reactions. | [https://universe.roboflow.com/allergen30/food_new-uuulf](https://universe.roboflow.com/allergen30/food_new-uuulf) | [https://link.springer.com/article/10.1007/s12161-022-02353-9](https://link.springer.com/article/10.1007/s12161-022-02353-9) | Ingredient Analysis | Open source |
| Food: allergen and allergy | A comprehensive list of food items with their corresponding allergies. | [https://www.kaggle.com/datasets/boltcutters/food-allergens-and-allergies](https://www.kaggle.com/datasets/boltcutters/food-allergens-and-allergies) |  | Ingredient Analysis | Open source |
| Food Ingredients and Allergens | The Food Allergens Dataset is a collection of information regarding allergens present in various food items. The dataset contains allergen information for a range of food ingredients, enabling the identification and analysis of potential allergens in different dishes and products. It serves as a valuable resource for researchers, food manufacturers, healthcare professionals, and individuals with food allergies. | [https://www.kaggle.com/datasets/uom190346a/food-ingredients-and-allergens/](https://www.kaggle.com/datasets/uom190346a/food-ingredients-and-allergens/) |  | Ingredient Analysis | Open source |
| [AllergyMap](https://pmc.ncbi.nlm.nih.gov/articles/PMC8075505/) | A corpus that maps free-text allergy mentions (medications, foods, etc.) in EHRs to standard terminologies (SNOMED, etc.) | [https://github.com/amywangmd/AllergyMap](https://github.com/amywangmd/AllergyMap) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC8075505/](https://pmc.ncbi.nlm.nih.gov/articles/PMC8075505/) | Text Mining | Open source |
| Allergen Status of Food Products | This dataset contains allergen status information for 400 food products, detailing ingredients, allergens present, pricing, and customer ratings, enabling allergen detection and analysis for researchers, manufacturers, and consumers | [https://www.kaggle.com/datasets/nandhanasuresh/allergen-status-of-food-products](https://www.kaggle.com/datasets/nandhanasuresh/allergen-status-of-food-products) |  | Allergen Identification, Ingredient Analysis, Labeling Compliance, Allergenicity Assessment | Open source |
| Ingredients with 16 Allergen Tags | This dataset lists 10,000 USDA ingredients, each tagged with 16 common allergen labels such as dairy, eggs, peanuts, gluten, and shellfish, with annotations indicating certainty or uncertainty of allergen presence. | [https://www.kaggle.com/datasets/khochawongwat/ingredients-with-17-allergen-tags](https://www.kaggle.com/datasets/khochawongwat/ingredients-with-17-allergen-tags) |  | Allergen Identification, Ingredient Analysis, Allergenicity Assessment | Open source |
| Food Ingredients and Allergens | This Food Allergens dataset contains 400 records detailing food products, their ingredients, associated allergens, and allergen presence prediction, supporting allergen detection and analysis for diverse applications. | [https://www.kaggle.com/datasets/uom190346a/food-ingredients-and-allergens](https://www.kaggle.com/datasets/uom190346a/food-ingredients-and-allergens) |  | Allergen Identification, Ingredient Analysis, Allergenicity Assessment, Text Mining | Open source |
| [ProPepper](https://pubmed.ncbi.nlm.nih.gov/26450949/) | ProPepper is a database of cereal prolamin epitopes, peptides and proteins for expert users that are dealing with protein chemistry, proteomics and mass spectrometry, method developments and related applications in food science, agricultural breeding or medical studies. | [https://ngdc.cncb.ac.cn/databasecommons/database/id/1686](https://ngdc.cncb.ac.cn/databasecommons/database/id/1686) | [https://pubmed.ncbi.nlm.nih.gov/26450949/](https://pubmed.ncbi.nlm.nih.gov/26450949/) | Ingredient Analysis, Product Development | Open source |
| Allergen Peptide Browser | Allergen Detection using Mass Spectrometry (MS) | [https://www.allergenpeptidebrowser.org/](https://www.allergenpeptidebrowser.org/) |  | Ingredient Analysis, Product Development | Open source |


## Patient Management & Clinical Decision Support

| Name | Description | Source | License | Tags | Access |
|------|-------------|--------|---------|------|--------|
| Food Allergy & Intolerance Dataset | This dataset contains data related to food allergies and intolerances. It includes key features such as age, gender, symptoms, food type consumed, IgE levels, and allergy history, helping in predictive modeling for food allergy detection and reaction severity assessment. | [https://datahub.io/@RuthvikUppala30/US-food-allergy-dataset](https://datahub.io/@RuthvikUppala30/US-food-allergy-dataset) |  | Severity Assessment, Risk Stratification | Open source |
| [DIABIMMUNE](https://pmc.ncbi.nlm.nih.gov/articles/PMC6361419/) | The DIABIMMUNE three-country cohort dataset tracks infants from Finland, Estonia, and Russia with similar HLA genetic risk for type 1 diabetes, collecting comprehensive longitudinal data including monthly stool microbiome sequencing (16S rRNA and whole-genome shotgun), clinical records, and lifestyle factors. It investigates the role of gut microbiome variations and early immune education in allergy and autoimmune disease development, providing valuable data to explore microbial and genetic influences on immune-related conditions | [https://diabimmune.broadinstitute.org/diabimmune/three-country-cohort](https://diabimmune.broadinstitute.org/diabimmune/three-country-cohort) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC6361419/](https://pmc.ncbi.nlm.nih.gov/articles/PMC6361419/) | Microbiome Analysis, Genetic Analysis | Open source |
| CFSAN Adverse Event Reporting System (CAERS) | The CFSAN Adverse Event Reporting System (CAERS) dataset contains approximately 90,000 reports of adverse events related to foods, dietary supplements, and cosmetics submitted to the FDA from 2004 to 2017. It includes detailed data on food products suspected in adverse reactions and associated symptoms, supporting analysis of patient risk factors, symptoms classification, and identification of allergenic ingredients in food products | [https://www.kaggle.com/datasets/fda/adverse-food-events](https://www.kaggle.com/datasets/fda/adverse-food-events) |  | Risk Stratification, Symptom Analysis, Ingredient Analysis | Open source |
| [DNA Methilation GSE59999](https://pubmed.ncbi.nlm.nih.gov/25678091/) | Genome wide DNA methylation profiling study of PBMC from 71 unique primary patient blood samples. The Illumina Human Methylation 450k array was used. 29 challenge proven food allergy, 29 sensitized but oral tolerant, 13 non food allergics Mixture of food allergy phenotypes (egg allergic (15), peanut allergic (14)), food sensitization phenotypes (egg sensitized (14), peanut sensitized (15)). 4 samples had technical replicate hybridzations. From "Blood DNA methylation biomarkers predict clinical reactivity in food-sensitized infants" | [https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59999](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59999) | [https://pubmed.ncbi.nlm.nih.gov/25678091/](https://pubmed.ncbi.nlm.nih.gov/25678091/) | Risk Stratification | Open source |
| [HealthNuts Study](https://www.nature.com/articles/s41597-022-01861-x#Sec6) | Mass citometry data from 36 participants encompassing non-allergic, peanut sensitized with tolerance, and clinically peanut allergic infants | [https://www.immport.org/shared/search?text=SDY2015](https://www.immport.org/shared/search?text=SDY2015) | [https://www.nature.com/articles/s41597-022-01861-x#Sec6](https://www.nature.com/articles/s41597-022-01861-x#Sec6) | Risk Stratification | Open source |
| [Dysfunctional Gut Microbiome Networks in Childhood IgE-Mediated Food Allergy](https://pmc.ncbi.nlm.nih.gov/articles/PMC7923212/#notes5) | To identify potential target microbes, which may play a key role in regulating/ influencing the microbe-microbe interactions, leading to the onset of food allergy. 16S rRNA, 33 allergic vs 27 controls | [https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA699997](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA699997) | [https://pmc.ncbi.nlm.nih.gov/articles/PMC7923212/#notes5](https://pmc.ncbi.nlm.nih.gov/articles/PMC7923212/#notes5) | Risk Stratification | Open source |
| [PEAR – Partners’ Enterprise-wide Allergy Repository (Food entries subset)](https://pmc.ncbi.nlm.nih.gov/articles/PMC4954633/) | Allergy entries from EHRs across a health system, with food terms extracted and normalized (approx. 158,552 food allergen records). |  | [https://pmc.ncbi.nlm.nih.gov/articles/PMC4954633/](https://pmc.ncbi.nlm.nih.gov/articles/PMC4954633/) | Risk Stratification | Gated |
| FSA Allergen Database Service (UK Nut allergy Registry) | Clinical and laboratory data from patients attending a UK allergy clinic for suspected nut allergy, including reaction history, severity, and lab tests. | [https://www.nal.usda.gov/research-tools/food-safety-research-projects/allergen-database-service](https://www.nal.usda.gov/research-tools/food-safety-research-projects/allergen-database-service) |  | Risk Stratification | Gated |
| Swiss legislation on food allergens data | This dataset is a German-language, hand-curated list of common food allergens based on Swiss legislation, compiled to support allergen identification and text matching for developers and researchers, with a focus on enabling structured, multilingual allergen data relevant to Switzerland | [https://github.com/foodopendata/food-allergens-ch](https://github.com/foodopendata/food-allergens-ch) |  | Allergen Identification, Ingredient Analysis, Labeling Compliance, Text Mining, Risk Stratification | Open source |

